metadata/src/main/java/sustain/synopsis/metadata/MetadataStarter.java

```diff
@@ -0,0 +1,50 @@
+package sustain.synopsis.metadata;
+
+import io.grpc.BindableService;
+import org.apache.log4j.Logger;
+import sustain.synopsis.dht.*;
+import sustain.synopsis.dht.zk.ZKError;
+
+
+import java.io.IOException;
+
+public class MetadataStarter {
+
+    public static Logger logger = Logger.getLogger(MetadataStarter.class);
+
+    public static void main(String[] args) {
+        if (args.length == 0) {
+            logger.error("Path to the configuration file is missing. Exiting!");
+            return;
+        }
+        String configFilePath = args[0];
+        logger.info("Using the configuration: " + configFilePath);
+
+
+        // initialize context
+        Context ctx = Context.getInstance();
+        try {
+            ctx.initialize(configFilePath);
```

Comment:
Do we need membership tracking in the metadata server? 

ZK is used for following reasons.
1. Used by DHT nodes to publish their location in the ring and get the neighbors (when we start supporting replication)
2. Used by Proxy nodes to get the ring info. Proxy nodes don't publish their info to ZK because it is not needed in a stateless design.